pelican/plugins/more_categories/more_categories.py

```diff
@@ -75,9 +74,24 @@ def create_categories(generator):
 
     generator.categories = sorted(
         list(cat_dct.items()),
-        reverse=generator.settings.get('REVERSE_CATEGORY_ORDER') or False,
+        reverse=generator.settings.get("REVERSE_CATEGORY_ORDER") or False,
     )
-    generator._update_context(['categories'])
+    generator._update_context(["categories"])
+
+    # Add subcategories
+    cats = {}
+    for category, articles in generator.categories:
+        for anc in category.ancestors:
+            if anc != category:
+                if anc.slug in cats:
+                    cats[anc.slug].append(category)
+                else:
+                    cats[anc.slug] = [category]
+    for category, articles in generator.categories:
+        if category.slug in cats:
+            category.subcategories = cats[category.slug]
+        else:
+            category.subcategories = []
```

Comment:
Could you split this up into two attributes, `children` and `descendents` (which includes the children)?

Also, please sort them to make things more convenient in the theme templates.

Reply:
For sure. Do you care which method of sort? Or just the normal `sort()` method on lists?